.github/workflows/acceptance_tests_common.yml

```diff
@@ -29,6 +29,7 @@ jobs:
               - 'testsuite/**'
               - '!.github/workflows/**'
               - '!testsuite/features/build_validation/**'
```

Comment:
Just a small remark to the line above your change. This did not work the last time I tried. Maybe we have some logic errors or other issues?!

Reply:
Maybe the order of these filters are important, maybe it just skip the rest of filters when one of them match, starting from top to bottom.